Agendas/23_01_27.md

```diff
@@ -0,0 +1,17 @@
+# Client Meeting January 27th, 2023. 
+# Title: Just a quick check in. 
+
+# Schedule
+*Teams will likely go in sequential order. Meeting shouldn't be longer than 30 minutes*
+
+# TEAM A Items
+ - Demo API and updated charactar animations
+ 
+
+# Questions
+- Processing times and recognizers. What should be the default option. (Preferably include both.) (Show the animations on discord)
```

Comment:
Don't forget to bring up the differences between the English processor and non English processor on English audio! We want to find out if we can ignore the pocket sphinx recognizer entirely and rely on the other one